app/src/main/res/layout/fragment_feed.xml

```diff
@@ -87,6 +87,26 @@
 
     </androidx.swiperefreshlayout.widget.SwipeRefreshLayout>
 
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:id="@+id/new_items_loaded_layout"
```

Comment:
Can't view nesting be avoided?

Reply:
This is indeed possible 👍 
Thanks for the hint.